### ISSUE TITLE
Switch to {{in-element}}

### ISSUE THREAD

**Issue by sandstrom** [yapplabs/ember-wormhole#114]
Would it make sense for this addon to switch to [{{in-element}})](https://github.com/emberjs/ember.js/pull/18867)?

Perhaps this addon serves another purpose and it doesn't make sense.

**Comment by mpanic** [comment]
That would be nice, but it's not "yet" possible to simple switch to `{{in-element}}` because it doesn't accept/render ember variables. 
`{{in-element}}` can only render plan HTML syntax.

**Comment by sandstrom** [comment]
@mpanic Are you sure?

These two articles seems to suggest otherwise:

https://balinterdi.com/blog/using-in-element-in-ember-js/
https://dev.to/shayanypn/ember-js-in-element-feature-273l

**Comment by nlfurniss** [comment]
@mpanic it seems to work. Here's [a demo](https://nlfurniss.github.io/ember-simple-wormhole/).

What do you think about investigating any feature gaps with this code and after any fixes, merging it in to this repo and releasing a v1?

**Comment by srsgores** [comment]
Yeah, and should this repo not be deprecated entirely, as we can almost replace `{{in-element}}` 1-1 with `ember-wormhole`?
